versioned_docs/version-4.0/getting-started/deploying-your-store.mdx

```diff
@@ -92,7 +89,7 @@ to configure Paperclip, just create an initializer like the following:
 ```ruby title="config/initializers/paperclip.rb"
 if Rails.env.production?
   Paperclip::Attachment.default_options.merge!(
-    storage: :s3,
+    storage: S3,
```

Comment:
I was reading the official guide [here](https://www.rubydoc.info/gems/kt-paperclip/Paperclip/Storage/S3) and it seems like it's expecting `:s3`. Can you help me understand where this `S3` is coming from?

Reply:
It looks like Rails adds some magic? https://guides.rubyonrails.org/active_storage_overview.html#setup I didn't try it myself, though, so we should make sure it's not a typo in the Rails guides.

Reply:
It could be the name of the adapter's class. Let me try to bring my friend @jeffreyguenther into the conversation, which is the author of [that PR on Rails guides](https://github.com/rails/rails/pull/31037). 😃

Reply:
Indeed, it's expecting :s3, which is a Ruby key-value hash."

Sorry, I mistakenly performed a global substitution on :s3. My intent was to change the one in the storage.config file, which is a YAML file, not a Ruby key-value hash. I will revert these mistakes.